src/viewer.js

```diff
@@ -3158,6 +3196,24 @@ function onCanvasPress( event ) {
         insideElementReleased: event.insideElementReleased,
         originalEvent: event.originalEvent
     });
+
+
+    gestureSettings = this.gestureSettingsByDeviceType( event.pointerType );
+    if ( gestureSettings.dblClickDragToZoom ){
+        var lastClickTime = THIS[ this.hash ].lastClickTime;
+        var currClickTime = $.now();
+
+        if ( lastClickTime === null) {
+            return;
+        }
+
+        if ((currClickTime - lastClickTime) < 2000) {
```

Comment:
Okay, good point! Clearly, 2000 is too big. I guess I thought the change there had been related to the setTimeout, but maybe it was just because you realized it was too big? 

At any rate, instead of being an explicit number, it should be `this.dblClickTimeThreshold`. 

Sorry about the confusion there!

Reply:
default value ?

Reply:
setting it as 1000